### PR TITLE
Vec ops docs

### PR DESCRIPTION
**Pull request by ImmanuelSegol**
## Describe the changes

This PR...

## Linked Issues

Resolves #

### CODE REVIEW

File: docs/docs/icicle/rust-bindings/vec-ops.md [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+}
+```
+
+#### Methods
+
+- **`add`**: Computes the element-wise sum of two vectors.
+- **`sub`**: Computes the element-wise difference between two vectors.
+- **`mul`**: Performs element-wise multiplication of two vectors.
+
+### Argument Validation
```
Comment (ChickenLover) [inline docs/docs/icicle/rust-bindings/vec-ops.md:108]: Remove this part:) `check_vec_ops_args` is just for internal use, and the user doesn't need to check anything before calling the methods

**Review by ChickenLover — CHANGES_REQUESTED** [review]
waiting for rust examples

File: docs/docs/icicle/rust-bindings/vec-ops.md (hunk 1 above)
Comment (ImmanuelSegol) [reply docs/docs/icicle/rust-bindings/vec-ops.md:108]: I wanted to explain that we implemented this for them, so developers wont think they need to implement it themselves

**Review by bigsky77 — APPROVED** [review]

File: docs/docs/icicle/rust-bindings/vec-ops.md (hunk 1 above)
Comment (ChickenLover) [reply docs/docs/icicle/rust-bindings/vec-ops.md:108]: Inputs validation is the default thing and we do it in all of our primitives, so I don't think we should mention it in the docs

**Pull request closed without merging**